probe/endpoint/conntrack_internal_test.go

```diff
@@ -13,6 +13,7 @@ import (
 )
 
 const conntrackCloseTag = "</conntrack>\n"
+const bufferSize = 1024 * 1024
```

Comment:
Have you tested this value in dev/prod? (to confirm the default doesn't value is large enough for at least our usecase)